xt-bundle-aos.cfg

```diff
@@ -0,0 +1,53 @@
+[path]
+# this shall be fast drive for doing most of the build on
+workspace_base_dir = /data/storage/work/xen_prod/build
+
+[bundle_config]
+top_template = bundle_templates/top_template.json
+boardModel = hw1
+formatVersion = 1
+components = dom0 domd domf
+ostree_path = /data/.aos_ostree
+bundle_deploy_dir = /data/bundles/
+
+[dom0]
+enabled = 1
+item_template = bundle_templates/item_template.json
+version = 1
+description = some description
+requiredVersion = 1
+minimalVersion = 1
+partition_no = 1
```

Comment:
Why it is needed for bundle generation?

Reply:
I'm using mk_sdcard_image script to generate bundle partitions. As long as full partition is needed for the rootfs update.

Reply:
Still unclear why it is needed for bundle generation?